src/components/CV_hierarchy/hierarchy.js

```diff
@@ -160,6 +160,15 @@ function CustomizedTreeView(props) {
             // add up the count from child labels
             totalCnt += countMap.get(childLabel); 
           }
+          // add itself label count
+          for (const idx in facetValues){
```

Comment:
The non-leaf labels were adding its child labels, but not itself's count, which lead some labels to be unappearing when `renderZeroCount` is false. Allow the non-leaf label's count to be added as well.